cmd/otelcol/config/collector/config.d.linux/properties.discovery.yaml.example

```diff
@@ -0,0 +1,18 @@
+# Copy this file to properties.discovery.yaml and add your uncommented discovery properties for use in discovery mode:
+# For more information:
+# https://github.com/signalfx/splunk-otel-collector/tree/main/internal/confmapprovider/discovery#discovery-properties
```

Comment:
Ideally we'll want to link to the official docs after launch, although this one is technical enough to perhaps not require it.